src/devices/Display/FontHelper14.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Provides segment mappings for hexadecimal digits and certain ASCII characters
+    /// </summary>
+    /// <remarks>
+    /// Sources:
+    /// Derived from /src/devices/Display/FontHelper.cs
+    /// </remarks>
+    public static class FontHelper14
```

Comment:
Consider changing this to internal visibility. Having this public seems to unnecessarily add "noise" to the API.

Reply:
Done.